src/main/scala/edu/ie3/powerFactory2psdm/config/ConfigValidator.scala

```diff
@@ -83,6 +87,13 @@ object ConfigValidator {
           s"The PV temperature correction factor (kT): ${params.kT} isn't valid. Exception: ${exc.getMessage}"
         )
     }
+    validateQCharacteristic(params.qCharacteristic) match {
+      case Success(_) =>
+      case Failure(exc) =>
+        throw ConversionConfigException(
+          s"The PV q characteristic configuration isn't valid. Exception: ${exc.getMessage}"
+        )
```

Comment:
```suggestion
        throw ConversionConfigException(
          "The PV q characteristic configuration isn't valid.", exc
        )
```

You can attach the whole Exception. This adds it to the head of the stack trace. If you just add the message, you can't "dive" down to where the issue really came from (bc you cut off the stacktrace).